docker/Dockerfile.ng

```diff
@@ -0,0 +1,213 @@
+FROM usualoma/ruby-with-therubyracer:2.4.0-alpine
```

Comment:
Currently, no. The [only image that exists is built on 2.4](https://hub.docker.com/r/usualoma/ruby-with-therubyracer/tags/)

That said, can definitely investigate whether the same technique would work on a 2.3.x build, and if so, we can definitely try it. Would be awesome to isolate the changes/differences required, and give us a far better platform for compatibility checking.

Have [opened an issue](https://github.com/usualoma/ruby-with-therubyracer/issues/2) to investigate this.